### DISCUSSION TITLE
Pipenv equivalent for pip options: --global-option=build_ext --global-option="-I/path/to/include/"

### DISCUSSION THREAD

**Discussion by mirstan** [pypa/pipenv#5610] [Q&A]
Hi,
Does anybody know the Pipenv equivalent for pip options: `--global-option=build_ext --global-option="-I/path/to/include/"` I'm trying to repeat a custom package build from source that works with `pip`. 

I'm not able to find any hints in the manual.

**Comment by kalebmckale** [comment]
You can always pass additional pip arguments using the `--extra-pip-args` command-line option to pass any pip-specific arguments. See https://pipenv.pypa.io/en/latest/advanced/
